hicommonwealth/commonwealth
# Updated thread patch to use the updated unified api

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/4791

## Description of Changes
- Now using the unified thread patch endpoint
- Removed extra logic/react-query code from the frontend (this was added in the #4545 migration from the old code)

## "How We Fixed It"
This was an enhancement, we improved the code by doing more with less. The frontend now uses the unified thread patch endpoint

## Test Plan
These actions on threads should work
1. Spam/Unspam
2. Pin/Unpin
3. Lock/Unlock
4. Update stage
5. Update topic
6. Update title/body
7. Add/remove collaborators

## Deployment Plan
N/A

## Other Considerations
- I noticed that maybe we didn't migrate `DELETE /linking/deleteLinks` and `POST /linking/addThreadLinks` to the unified API, cc @rbennettcw to confirm?
- We can update the dangerous thread cache file to be a bit more simpler.
